source/PokerSetup.py

```diff
@@ -0,0 +1,344 @@
+#   Code still needs to incorporate a pot.
+
+from source.CardGames import *
+import sys
+
+PlayersStillInGameList = PlayerList
+BlindsList = PlayersStillInGameList
+PlayersPlayingRoundList = PlayersStillInGameList
+PlayersOutOfGameList = []
+
+class PokerGameSetup:
+
+    def __init__(self, name, money: int = 0):
+        self.name = []
+        self.hand = []
+        self.money = money
+
+    def preGame(self, players: PlayerList):
+        #   Give each player their money.
+        #
+        #   Option to have each player start with the same amount of money,
+        #   or with various amounts of money.
+        try:
+            Same = input('''Enter "Same" if each player starts with the same amount.
+Enter "Different" if each player start with different amounts: ''')
+
+            #   Each player starts with the same amount.
+            if Same == "Same" or Same == "same" or Same == "SAME" or Same == "S" or Same == "s":
```

Comment:
You can use string functions like .lower() to help. That function converts the whole string to lower case, so you could just check "Same.lower() == 'same'" instead of trying all different capitalizations